src/pages/LandingPage.tsx

```diff
@@ -25,14 +25,22 @@ const NoShows = styled.div`
   top: 1rem;
 `;
 
+const getDateString = () => {
```

Comment:
we have ./src/utils/getTodayDateISO.ts , please use that function getTodayDateISO() to get ISO date